### PR TITLE
Add city service nunit tests

### PR DESCRIPTION
**Pull request by tetyanakutsko**

### CODE REVIEW

**Review by WanDER33 — APPROVED** [review]

**Review by MykhailoKlapchuk — APPROVED** [review]

File: EPlast/EPlast.Tests/Services/City/CityMembersServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EPlast.Tests.Services.City
+{
+    class CityMembersService
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/City/CityMembersServiceTests.cs:7]: !????

File: EPlast/EPlast.Tests/Services/City/CityServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+            var result = await _cityService.GetAllDTOAsync();
+
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.IsInstanceOf<IEnumerable<CityDTO>>(result);
+        }
+
+        [Test]
+        public async Task GetCitiesByRegionAsync_ReturnsCities()
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/City/CityServiceTests.cs:89]: Return or returns?  
I sing where you might have to use some rules.
You may use return or returns.
But here is one rule to use one of  them.

**Review by voroniak — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.Tests/Services/City/CityMembersServiceTests.cs (hunk 1 above)
Comment (tetyanakutsko) [reply EPlast/EPlast.Tests/Services/City/CityMembersServiceTests.cs:7]: Thanks. Was aded by mistake. Deleted already

File: EPlast/EPlast.Tests/Services/City/CityServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+
+            for (int i = 0; i < count; i++)
+            {
+                cities.Add(new DataAccessCity.City());
+            }
+
+            return cities.AsQueryable();
+        }
+
+        public IQueryable<CityDTO> CreateFakeCityDto(int count)
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/City/CityServiceTests.cs:288]: it seems to me that the methods that return test data in tests should be private

File: EPlast/EPlast.Tests/Services/City/CityServiceTests.cs [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+        public Region GetTestRegion()
+        {
+            var region = new Region()
+            {
+                ID = 1,
+                RegionName = "Lviv",
+                Description = "Lviv region"
+            };
+
+            return region;
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/City/CityServiceTests.cs:359]:
```suggestion
            return new Region()
            {
                ID = 1,
                RegionName = "Lviv",
                Description = "Lviv region"
            };
\
```

File: EPlast/EPlast.Tests/Services/City/CityServiceTests.cs [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+
+        public DataAccessCity.City GetTestNewCity()
+        {
+            var city = new DataAccessCity.City
+            {
+                Name = "city",
+                Logo = "710b8b06-6869-45db-894f-7a0b131e6c6b.jpg"
+            };
+
+            return city;
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Services/City/CityServiceTests.cs:370]:
```suggestion
           return new DataAccessCity.City
            {
                Name = "city",
                Logo = "710b8b06-6869-45db-894f-7a0b131e6c6b.jpg"
            };

```

Comment (tetyanakutsko) [reply EPlast/EPlast.Tests/Services/City/CityServiceTests.cs:370]: Thanks a lot!!!
It's changed.

**Review by voroniak — APPROVED** [review]
LGTM!

**Pull request merged**